app/downstream_payload.rb

```diff
@@ -52,27 +44,40 @@ def unpublishing
     edition.unpublishing
   end
 
-  def content_payload
-    Presenters::EditionPresenter.new(
-      edition, draft: draft
-    ).for_content_store(payload_version)
+  def content_presenter
+    Presenters::EditionPresenter.new(edition, draft: draft)
+  end
+
+  def redirect_presenter
+    RedirectPresenter.from_edition(edition)
+  end
+
+  def gone_presenter
+    GonePresenter.from_edition(edition)
   end
 
-  def redirect_payload
-    RedirectPresenter.present(
-      base_path: base_path,
-      publishing_app: edition.publishing_app,
-      public_updated_at: unpublishing.created_at,
-      redirects: unpublishing.redirects,
-    ).merge(payload_version: payload_version)
+  def vanish_presenter
+    VanishPresenter.from_edition(edition)
   end
```

Comment:
This is a good point, but I think it should be for a separate card. I'll create one.